src/redux/books/books.js

```diff
@@ -0,0 +1,20 @@
+const ADDBOOK = {
+  type: 'ADD',
+};
+
+const REMOVEBOOK = {
+  type: 'REMOVEBOOK',
+};
+
+const bookReducer = (state = [], action) => {
+  switch (action.type) {
+    case ADDBOOK:
+      return [...state];
+    case REMOVEBOOK:
+      return state;
+    default:
+      return state;
+  }
+};
```

Comment:
- [ ] For the reducer, when the `action.type` is `ADDBOOK`, you should return the `state` and the book which is the `payload`. That is append the payload to the state. `[...state, action.payload]`. When the `action.type` is REMOVEBOOK, you should filter the book with the `id` as `payload` from the state. You can use the `filter` method (preferable) here or any method you can use to remove an item from an array. 👍 💻